releases/release-1.26/release-notes/maps/pr-111973-map.yaml

```
@@ -0,0 +1,5 @@
+pr: 111973
+releasenote:
+  text: Deprecated beta APIs scheduled for removal in `v1.26` are no longer served.
+    See https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-26 for
```

Use `[]()` markdown syntax for links:

```suggestion
    See [deprecation guide for 1.26](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#v1-26) for
```